addons/l10n_in_ewaybill_stock/tests/test_ewaybill_stock.py

```diff
@@ -198,3 +199,10 @@ def test_ewaybill_stock_test_3(self):
         }
         ewaybill._l10n_in_ewaybill_stock_handle_zero_distance_alert_if_present(response)
         self.assertEqual(ewaybill.distance, expected_distance)
+
+    def test_ewaybill_stock_empty_product_hsn(self):
```

Comment:
```suggestion
    def test_ewaybill_stock_4(self):
    """Test with missing hsn code"""
```
please keep the method name consistent. I haven't debug it yet but the picking name order messes up